src/node.hpp

```diff
@@ -11,6 +11,10 @@
 #include <string>
 #include <thread>
 #include <vector>
+#include <sys/stat.h>
+
+/* Function prototype */
```

Comment:
not so useful comment.
/* Create log dir if not exists */

Reply:
may be even avoid this. and create a static function. I assume it is not used outside this file..

Reply:
Done